HarpGuru/Controls/CovarianceButtonList/index.ts

```diff
@@ -0,0 +1,2 @@
+export { CovarianceButtonList } from './components'
+export type { CovarianceButtonListProps } from './types'
```

Comment:
I should probably come up with a better name for these buttons, but these are not permanent. A much better UI component needs to be developed. The persistent part of this merge request will be the CovarianceSeries module so the naming is important there.

Reply:
https://github.com/js-jslog/harpnative/issues/12 raised